drivers/syslog/vsyslog.c

```diff
@@ -132,8 +133,29 @@ int nx_vsyslog(int priority, FAR const IPTR char *fmt, FAR va_list *ap)
 #if defined(CONFIG_SYSLOG_TIMESTAMP)
   /* Pre-pend the message with the current time, if available */
 
+#if defined(CONFIG_SYSLOG_TIMESTAMP_FORMATED)
+  time_t time;
+  struct tm * tm;
+  char date_buf[CONFIG_SYSLOG_TIMESTAMP_BUFFER];
```

Comment:
move to the begin of the function per coding style.

Reply:
Done.